08-plot-ggplot2.Rmd

```diff
@@ -35,16 +35,16 @@ package, and the [ggplot2][ggplot2] package.
 [lattice]: http://www.statmethods.net/advgraphs/trellis.html
 [ggplot2]: http://www.statmethods.net/advgraphs/ggplot2.html
 
-Today we'll be learning about the ggplot2 package, because
+Today we'll be learning about the **ggplot2** package, because
```

Comment:
Back quotes (to put in typewriter case) rather than bold, since it's a library name.